frontend/src/components/grantaccess/GrantDelegatedAdmin.vue

```diff
@@ -89,15 +99,93 @@ const areVerificationsPassed = () => {
     );
 };
 
-const handleSubmit = async () => {
-    // This will be implemented in task #1160
-    console.log('Data to send to backend', formData.value);
+const confirmSubmit = async () => {
+    const username = formData.value.userId.toUpperCase();
+    const role = getSelectedRole()?.name;
+    const successList: string[] = [];
+    let errorList: string[] = [];
+    let errorCode = ErrorCode.Default;
+    const data = toRequestPayload(formData.value);
 
+    try {
+        const returnResponse =
+            await AdminMgmtApiService.delegatedAdminApi.createAccessControlPrivilegeMany(
+                data
+            );
+        returnResponse.data.forEach((response) => {
```

Comment:
The POST delegated admin method returns a list of result for each forest client numbers as described in ticket #1089.  We need to check the response status for each forest client number.